src/install.rst

```diff
@@ -17,225 +17,280 @@ When completed, you'll be ready to run Nextstrain :term:`workflows <workflow>`.
 
-         .. group-tab:: Ubuntu Linux
 
-            .. note:: Steps for other Linux distributions (Debian, CentOS, RHEL, etc.) will be similar, though specific commands may vary slightly.
+            .. group-tab:: Windows
 
-            1. Install Docker Engine using the standard Ubuntu package:
+               `Install Windows Subsystem for Linux (WSL) 2`_.
+               You may have to restart your machine when configuring WSL.
 
-               .. code-block:: bash
+               `Install Docker Desktop for Windows <https://docs.docker.com/desktop/install/windows-install/>`_ with the `WSL 2 backend <https://docs.docker.com/desktop/windows/wsl/>`_.
 
-                  sudo apt install docker.io
+
+            .. group-tab:: WSL on Windows
+
+               `Install Docker Desktop for Windows`_ with the `WSL 2 backend`_.
 
                .. note::
 
-                  See `Docker's installation documentation <https://docs.docker.com/engine/install/ubuntu/>`__ for alternative installation methods.
+                  Make sure to follow through to the **Enabling Docker support in WSL 2 distros** section and the last step of enabling **WSL Integration**.
+                  If you forget to do this, ``docker`` won't work in the WSL terminal.
+
+               .. include:: snippets/wsl-home-dir.rst
+
 
-            2. Add your user to the `docker` group:
+            .. group-tab:: Ubuntu Linux
 
-               .. code-block:: bash
+               .. note:: Steps for other Linux distributions (Debian, CentOS, RHEL, etc.) will be similar, though specific commands may vary slightly.
 
-                  sudo gpasswd --add $USER docker
+               Install Docker Engine using the standard Ubuntu package:
 
-            3. Restart your machine.
-            4. Install the Nextstrain CLI:
+                  .. code-block:: bash
 
-               .. code-block:: bash
+                     sudo apt install docker.io
 
-                  curl -fsSL --proto '=https' https://nextstrain.org/cli/installer/linux | bash
+               or see `Docker Engine's installation documentation <https://docs.docker.com/engine/install/ubuntu/>`__ for alternative methods.
 
-            5. Follow the installer's final instructions to setup your shell config.
+               Add your user to the ``docker`` group:
+
+                  .. code-block:: bash
+
+                     sudo gpasswd --add $USER docker
+
+               Log out and back in again for the group change to take effect.
+
+
+      2. Set up the runtime:
+
+         .. code-block:: none
+
+            nextstrain setup --set-default docker
+
+
+   .. group-tab:: Conda
+
+      .. note::
+
+         Due to installation constraints, there is no way to use Nextstrain's Conda runtime on Windows directly. Starting from the beginning, follow steps for **WSL on Windows** if the Conda runtime is desired, or use the **Docker** runtime instead.
```

Comment:
Non-Windows users might be confused by this message. I think it's unnecessary to show for those users, and adding tabs here to be consistent with other runtime tabs would make for a better user experience. Example: 2c5f4d612b0d57097bda0b03487aaba07ec0c7da

Reply:
Good point. I avoided tabs here to try and simplify the appearance, but looking at your example, I agree it's better. Applied 2c5f4d612b0d57097bda0b03487aaba07ec0c7da as c2e954c977df72fffbd3fbf4c57eb4c50778cb74 after fixing indentation and adding your comment above as the commit message.